src/main/java/com/pet/domains/post/service/MissingPostBookmarkService.java

```diff
@@ -30,11 +32,29 @@ public void createMissingPostBookmark(Long postId, Account account) {
                 .missingPost(getMissingPost)
                 .build()
         );
+        OptimisticLockingHandlingUtils.handling(
+            getMissingPost::increaseBookCount,
+            5,
+            "실종/보호 게시글 북마크 증가"
+        );
     }
 
     @Transactional
     public void deleteMissingPostBookmark(Long postId, Account account) {
-        missingPostBookmarkRepository.deleteByMissingPostIdAndAccount(postId, account);
+        MissingPost getMissingPost = missingPostRepository.findById(postId)
+            .orElseThrow(ExceptionMessage.NOT_FOUND_MISSING_POST::getException);
+
+        Long resultCount = missingPostBookmarkRepository.deleteByAccountAndMissingPost(account, getMissingPost);
+        if (Objects.nonNull(resultCount) && resultCount.intValue() == 1L) {
```

Comment:
요거 제가 케빈한테 피드백을 제대로 안달아서 생겼군여 원래 resultCount.inValue() == 1 이였는데 다음과 같이 수정해도됩니당!
```suggestion
        if (Objects.nonNull(resultCount) && resultCount == 1L) {
```

Reply:
감사합니다!!

Reply:
zzzz 그거였군용..